### PR TITLE
Add IssueViz.Security assembly

### PR DESCRIPTION
**Pull request by rita-g-sonarsource**
Part of #1809

### CODE REVIEW

File: src/Integration.Vsix/Manifests/VS2017/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
-        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="Integration.TeamExplorer" Path="|Integration.TeamExplorer|" />
-        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="IssueViz" Path="|IssueViz|" />
-      <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="IssueViz" Path="|IssueViz;PkgdefProjectOutputGroup|" />
-    </Assets>
-    <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.Compiler" Version="[15.0,16.0)" DisplayName="C# and Visual Basic Roslyn compilers" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.Static.Analysis.Tools" Version="[15.0,16.0)" DisplayName="Static analysis tools" />
-    </Prerequisites>
+  <Metadata>
```
Comment (rita-g-sonarsource) [inline src/Integration.Vsix/Manifests/VS2017/source.extension.vsixmanifest:3]: My editor did whitespace changes...

File: src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt [hunk 2]
```diff
@@ -238,6 +238,18 @@ Referenced assemblies:
 - 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 # Number of references: 26
 
+---
+Assembly: 'SonarLint.VisualStudio.IssueVisualization.Security, Version=4.28.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+Relative path: 'SonarLint.VisualStudio.IssueVisualization.Security.dll'
+
+Referenced assemblies:
+- 'Microsoft.VisualStudio.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```
Comment (duncanp-sonar) [inline src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt:246]: This is the version we don't want, isn't it?

Comment (rita-g-sonarsource) [reply src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt:246]: Yes, I'll remove it in the future PR

**Review by duncanp-sonar — APPROVED** [review]
I'm guessing this is just the empty project, and that it's not supposed to produce a VS-version-specific assembly yet? If so, looks good.

**Comment by rita-g-sonarsource** [comment]
@duncanp-sonar yes, this is just the empty project boilerplate, the versioning will be in a different PR

**Pull request merged**